src/sentry/api/endpoints/organization_releases.py

```diff
@@ -623,13 +624,14 @@ def post(self, request: Request, organization) -> Response:
                 status = 201
 
             analytics.record(
-                "release.created",
-                user_id=request.user.id if request.user and request.user.id else None,
-                organization_id=organization.id,
-                project_ids=[project.id for project in projects],
-                user_agent=request.META.get("HTTP_USER_AGENT", ""),
-                created_status=status,
-                auth_type=get_auth_api_token_type(request.auth),
+                ReleaseCreatedEvent(
+                    user_id=request.user.id if request.user and request.user.id else None,
+                    organization_id=organization.id,
```

Comment:
`organization` is typed as `Any` in this function, so it is difficult to tell whether this is safe